Makefile

```diff
@@ -1,8 +1,23 @@
+export GO111MODULE=on
+
 vet:
 	@go vet -copylocks ./...
 
 test-unit:
 	@go test -short ./...
 
+test-system:
+	@(cd test/system && make)
```

Comment:
Nit: `$(MAKE) -C subdir` or `cd test/system && $(MAKE)` so `make -n` works.

Reply:
And I don't anticipate putting anything else in test/, so I'm for putting this stuff in test/ instead of test/system/.